chpl-checkstyle-checks.xml

```diff
@@ -68,6 +68,7 @@
         <module name="JavadocMethod">
             <property name="severity" value="warning"/>
             <property name="scope" value="public" />
+	    <property name="minLineCount" value="10" />
```

Comment:
This is still misaligned

Reply:
I tried editing with different editors but at my end, the alignment looks good. Can you please help me out with this?